wp-content/themes/core/assets/css/src/theme/layouts/default.pcss

```diff
@@ -81,6 +81,16 @@ main {
 			width: calc(100% - 2 * var(--grid-gutter));
 		}
 	}
+
+	& > .wp-block-columns {
+		width: calc(100% - 2 * var(--grid-gutter-small));
+		margin-left: auto;
+		margin-right: auto;
+
+		@media (--viewport-medium) {
+			width: calc(100% - 2 * var(--grid-gutter));
```

Comment:
Probably should be `--grid-gutter-medium` and then move `grid-gutter` to a viewport-full media query as well.

Reply:
@dpellenwood I got this from line 81 above, where we are also using `-grid--gutter` at the medium size viewport. It looks like that may be a mistake. Do we want to go ahead update both of these?

Reply:
Updated!